packaging/rhel/syslog-ng.spec

```diff
@@ -46,26 +50,39 @@ Source4: syslog-ng.logrotate7
 %bcond_with java
 %endif
 
+%global glib2_ver 2.32
```

Comment:
Let's discuss this on today's meeting: IMHO this wouldn't solve the problem Evan told us: if the rpm package built with glib 2.56, but on the system there is a version (e.g. 2.46) that satisfies the min_version, but misses symbols that 2.56 has it won't work.

I think the version has to be the same as the one used for the rpm creation (in this case 2.56.1).

What is interesting here is that on the given system the required glib version (2.56.1) is available, but the system is not upgraded yet.
As far as I understood Evan, he said if the versioning is okay in the spec file, then during install it would search for the 2.56.1 version of glib.

Reply:
Yeah, my planned workaround for the problem was this simple:
```
# Copr always uses the latest RHEL release to build syslog-ng
# starting syslog-ng might fail on older RHEL releases
# change this if you build on an older RHEL version
# can be as low as 2.32
BuildRequires: glib2-devel >= 2.54
```